apps/camera/js/app.js

```diff
@@ -274,6 +273,7 @@ App.prototype.loadLazyModules = function() {
 
   this.loadL10n(done());
   this.loadLazyControllers(done());
```

Comment:
Maybe the naming isn't clear here. The `all-done` util returns a function that when called returns a new callback function. Each time the function is called it counts 1. The callback function passed on line `279` line be called until the three callback above have fired.

Soz finding it really hard to explain! Check out the new test I wrote as part of this patch in `app_test.js`.